growth/too/tasks/ref.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import subprocess
+import shlex
+from subprocess import PIPE
+import numpy as np
+import tempfile
+import requests
+
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from ..flask import app
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_references',)
+
+
+@app.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_references():
+
+    user = app.config['IRSA_USERNAME']
+    pw = app.config['IRSA_PASSWORD']
+
+    if not 'IRSA_USERNAME' in app.config or not 'IRSA_PASSWORD' in app.config:
+        log.info('Missing IRSA_USERNAME or IRSA_PASSWORD...')
+        return
+
+    params = (('josso_cmd', 'login'),
+              ('josso_username', app.config['IRSA_USERNAME']),
+              ('josso_password', app.config['IRSA_PASSWORD']))
+
+    session = requests.Session()
+    session.get('https://irsa.ipac.caltech.edu/account/signon/login.do', params=params, verify=False)
+    params = (('WHERE', 'nframes>=15 AND startobsdate>=\'2018-02-05T00:00:00Z\''),)
+    r = session.get('https://irsa.ipac.caltech.edu/ibe/search/ztf/products/ref', params=params, verify=False)
```

Comment:
Please use pyvo.dal.TAP rather than raw requests so that it will parse the output for you.

```python
results = get_tap_client().search("""
    SELECT field, ccdid, qid, fid, maglimit FROM ztf.ztf_current_meta_ref
    WHERE nframes >= 15 AND startobsdate >= '2018-02-05T00:00:00Z'
""").to_table()
```